README.md

```diff
@@ -1,60 +1,43 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Enumerable
 
-> Description the project.
+> This project represents our own implementation of the all?, any? and filter methods 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby
+- IRB
+- linter: Rubocop
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+```bash
+  git clone git@github.com:joel-kalema/my_Enumeration.git
+  cd into the folder by typing: cd my_Enumeration
+```
+open your code editor and run 
+```bash
+  bundle install
```

Comment:
- [ ] I would recommend including the steps to run the application as well. [Hint: `ruby my_list.rb`]